ember-container-query/rollup.config.mjs

```diff
@@ -0,0 +1,75 @@
+import { Addon } from '@embroider/addon-dev/rollup';
+import copy from 'rollup-plugin-copy';
+import typescript from 'rollup-plugin-ts';
+
+const addon = new Addon({
+  srcDir: 'src',
+  destDir: 'dist',
+});
+
+export default {
+  // This provides defaults that work well alongside `publicEntrypoints` below.
+  // You can augment this if you need to.
+  output: addon.output(),
+
+  plugins: [
+    // These are the modules that users should be able to import from your
+    // addon. Anything not listed here may get optimized away.
+    addon.publicEntrypoints([
+      'components/container-query.js',
+      'helpers/aspect-ratio.js',
+      'helpers/cq-aspect-ratio.js',
+      'helpers/cq-height.js',
+      'helpers/cq-width.js',
```

Comment:
These helpers have been deprecated and will be removed in a separate pull request.